apps/hash-graph/lib/graph/src/api/rest/utoipa_typedef.rs

```diff
@@ -19,9 +19,15 @@ pub enum ListOrValue<T> {
 // Utoipa doesn't seem to be able to generate sensible interfaces for this, it gets confused by
 // the generic
 impl<T> ListOrValue<T> {
-    pub(crate) fn generate_schema(schema_name: &'static str) -> RefOr<Schema> {
+    pub(crate) fn generate_schema(schema_name: &'static str, replace: bool) -> RefOr<Schema> {
```

Comment:
Would you mind replacing the boolean with an enumeration?

Reply:
fixed in [4e2f303](https://github.com/hashintel/hash/pull/2718/commits/4e2f3033506bbf26a069f1964e613f0b3b43bf0c)

Reply:
Cool, thanks. Now make it `Copy`, pass it by value, and I'll approve 🪨

Reply:
yes sir 🫡 [d007743](https://github.com/hashintel/hash/pull/2718/commits/d007743d7f466d2fb3d8744730dca548b9e1d623)